component/vdr/trustbloc/vdr_test.go

```diff
@@ -1500,7 +1508,7 @@ func Test_canonicalizeDoc(t *testing.T) {
     },
     "did:example:123456789abcdefghi#keys-3"
   ],
-  "@context": ["https://w3id.org/did/v1"],
```

Comment:
I'm not sure this change is right. If there is a requirement to process DID docs with this context then it should be left that way.

Reply:
according to [DID Doc V1 Spec](https://www.w3.org/TR/did-core/) contextV1 is `https://www.w3.org/ns/did/v1`

cc @troyronda